src/platform/packages/shared/kbn-esql-validation-autocomplete/src/definitions/commands.ts

```diff
@@ -151,10 +151,11 @@ const statsValidator = (command: ESQLCommand) => {
       // * the agg function is at root level
       // * or if it's a operators function, then all operands are agg functions or literals
       // * or if it's a eval function then all arguments are agg functions or literals
+      // * or if a named param is used
       function checkFunctionContent(arg: ESQLFunction) {
```

Comment:
Quick question: What is the motivation for defining `checkFunctionContent` and other functions inside `statsValidator`? Shall we move outside to the top level?

Reply:
yes this makes sense, will taje a look tmr in a follow up PR